src/pages/Selector/container.jsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
+import { transformEmployees as transform } from './transform-employees';
 
-// temporarily disabling this rule until the function is fleshed out
-/* eslint-disable react/prefer-stateless-function */
 const SelectorContainer = Wrapped =>
-  class Selector extends React.Component {
+  class extends React.Component {
+    state = { employees: [] };
+
+    componentDidMount() {
+      this.fetchEmployees();
+    }
+
+    fetchEmployees = () => {
+      fetch('http://localhost:8081/api/users').then(resp => {
```

Comment:
I tend to drive API endpoints from constants - makes it easier for reuse and refactoring. 

It might even make more sense to add a userService.js? 

Means we can reuse elsewhere.